exporter/signalfxexporter/translation/constants.go

```diff
@@ -389,6 +193,7 @@ translation_rules:
   - state
 
 # convert memory metrics
+# memory.used needed to calculate memory.utilization
```

Comment:
I believe we don't require any of the other translations here in the exporter, right? We need `memory.used` to calculate the utilization, but otherwise these would be mapped post ingestion right?

Reply:
Right. I guess we could copy the metric and split it for only the single dimension instead of translating them all

Reply:
Yes, and we should also `drop` `memory.used` after the calculation since it'll be redundant.

Reply:
Sound good to me, I'll update